flask_image_resizer/core.py

```diff
@@ -583,19 +617,22 @@ def get_aspect_ratio(path, filename):
         return None
 
 
-def get_image_format(path: str) -> str:
+def get_image_format(path: str, query) -> str:
     """
     We return the format from Pillow.
 
     If there is an issue with Pillow, we return 'jpeg'.
     """
     try:
         # Try to get format from PIL first
-        image = Image.open(path)
-        return image.format.lower()
-    except Exception as e:
-        current_app.logger.error(f'Exception: {e}')
-        return 'jpeg'
+        with Image.open(path) as image:
+            return image.format.lower()
+    except Exception:
+        image_format = 'svg+xml'
+        if not path.endswith('.svg'):
+            image_format = (query.get('format', '') or os.path.splitext(path)[1][1:] or 'jpeg').lower()
+            image_format = {'jpg': 'jpeg'}.get(image_format, image_format)
+        return image_format
```

Comment:
I'm adding back this on exception because sometimes PIL is not able to give me the format and it fails. This happened with an .svg

This is similar but solutions didn't work: https://stackoverflow.com/q/19230991